### ISSUE TITLE
Faulty symlink when using WinGet

### ISSUE THREAD

**Issue by headquarter8302** [Genymobile/scrcpy#4027]
- [x] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [x] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Windows 10
 - scrcpy version: 2.0
 - installation method: winget
 - device model: Lenovo P1M
 - Android version: 5.1

**Describe the bug**
After installing through winget, a symlink is generated at `%LOCALAPPDATA%\Microsoft\WinGet\Links` as a mean of getting the executable into Path. However, when executing the symlink through the terminal (PowerShell 7.3.4 Core on Win10), it doesn't execute anything, just outputting nothing and returning to prompt. Even though manually going through the directory using File Explorer and opening it by double-clicking will launch it as normal.

```ps
 ASUS  PS  Monday 09:20:08 PM
┌ C:\Users\ASUS
└ ❯ scrcpy

 ASUS  PS  Monday 09:20:12 PM
┌ C:\Users\ASUS
└ ❯
```

And no, the custom PS profile isn't interfering with anything. Heck, running with `-noprofile` will return an error instead, as if there was no symlink at all.

```ps
PowerShell 7.3.4
PS C:\Users\ASUS> scrcpy
scrcpy: The term 'scrcpy' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
PS C:\Users\ASUS>
```

However, executing the command via cmd.exe results in a stream of error prompts instead, reading: "The code execution cannot proceed because [multiple DLLs] was not found. Reinstalling the program may fix this problem."

**Comment by rom1v** [comment]
Please report to WinGet.

**Comment by headquarter8302** [comment]
WinGet repo confirmed that this is an upstream issue with their app not being able to handle portable executables with DLLs

**Comment by rom1v** [comment]
> their app not being able to handle portable executables with DLLs

Then they should probably build a non-portable version and use the dependencies (ffmpeg, sdl, libusb) from their repo: https://github.com/Genymobile/scrcpy/blob/958f22490b3ab86677bd8b6126ddaa4de4082798/meson_options.txt#L4

**Comment by BurtHarris** [comment]
Yup, just had the same problem.   You might consider removing the mention of winget from 
/windows.md

PS: Thanks, _scrcpy_ sounds like just what I was looking for.

**Comment by headquarter8302** [comment]
Upstream solution available in Winget 1.9 clients

https://github.com/microsoft/winget-pkgs/issues/107858#issuecomment-2515716218

**Comment by aisgbnok** [comment]
This has been fixed for all versions of the `Genymobile.scrcpy` package in [microsoft/winget-pkgs](https://github.com/microsoft/winget-pkgs), including the latest `3.1` release of scrcpy. scrcpy binaries (dlls, etc.) are now properly added to path winget.
